product_status/tests/test_product_status.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from freezegun import freeze_time
+
+from odoo.exceptions import ValidationError
+from odoo.tests import Form, tagged
+
+from odoo.addons.product.tests.common import TestProductCommon
+
+
+@tagged("post_install", "-at_install")
+class TestProductStatusCase(TestProductCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
```suggestion
```
As base class `TestProductCommon` now inherits from `BaseCommon`, you don't need to disable the tracking.